doc/conf.py

```diff
@@ -41,16 +41,16 @@
 
 # General information about the project.
 project = u'Nextcloud Client Manual'
-copyright = u'2013-2020, The Nextcloud developers'
+copyright = u'2013-2021, The Nextcloud developers'
 
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
 # The short X.Y version.
-version = '3.0'
+version = '3.2'
 # The full version, including alpha/beta/rc tags.
-release = '3.0'
+release = '3.2.1'
```

Comment:
@mgallien Shouldn't it be 3.2.`0`?

Reply:
And I will add that step into the release process issues until #3210 is merged